### DISCUSSION TITLE
Список предупреждений по пользователю не отображается при слишком длинных описаниях последних предупреждений

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#3] [Общее]
В команде `/violations` присутствует баг. Если участнику выдавать предупреждения с длинными описаниями, то, при выполнении команды `/violations` по вышеупомянутому участнику есть вероятность, что сообщение не будет отправлено из-за того, что эти предупреждения попадают в "Последние 10" и превышают лимит символов на поле встроенного сообщения (предположительно 1024 символа). При этом в интерфейсе отображается лишь стандартное сообщение "Lacuna думает", а в логах, скачанных из панели, вообще не упоминается использование команды. То есть пользователь, не знающий о лимитах дискорда, не может понять, почему команда не работает.
![image](https://github.com/LacunaHub/.github/assets/121901513/6ebc0494-03f3-47fb-b6f7-ef182d778337)
![image](https://github.com/LacunaHub/.github/assets/121901513/faf135b9-1298-4924-9f7d-c146c58ee512)
На первом скриншоте: имелось 5 предупреждений с коротким описанием и ещё 5 с длинным. Команда работает.
На втором скринште все 10 последних предупреждений с длинным описанием, "Lacuna думает".

**Comment by rovius** [comment]
В качестве варианта решения проблемы предлагаю разбить последние предупреждения по отдельным полям, в заголовок указывать временную метку и идентификатор, а в описание - само описание предупреждения. Да, сообщения получится больше в размерах, но это не сильно большая проблема, так как оно эфемерное

**Reply by danliyev** [reply]
Спасибо

Исправление будет в релизе v4.11.3

![Discord_1GxqgYGJua](https://github.com/LacunaHub/.github/assets/30476012/d91fade6-8feb-48de-a26f-77ee2066cf2d)

**Comment by danliyev** [comment]
Исправлено в версии 4.11.3
